echopype/convert/api.py

```diff
@@ -90,7 +96,7 @@ def _validate_path(
             out_path = out_dir / (root.stem + file_format)
 
     # Create folder if save_path does not exist already
-    fsmap = fsspec.get_mapper(str(out_dir), **output_storage_options)
+    fsmap = fsspec.get_mapper(out_dir.as_posix().replace(':/', '://'), **output_storage_options)
```

Comment:
This is really hacky, I think there should be a check for `://` instead higher up in the function. If the output string has `://` It's a sign that this is not a local path, so don't use pathlib to parse the save_path string, but rather keep it the way it is and use fsspec.